plugins/api/src/switch_user.cpp

```diff
@@ -171,6 +209,15 @@ namespace
                 return SYS_PROXYUSER_NO_PRIV;
             }
 
+            // Verify there are no open L1 descriptors. Return an error if there are. This forces the
+            // client to clean up behind themselves and keeps this API plugin from becoming overly complicated.
+            for (const auto& l1d : std::span{L1desc}) {
+                if (FD_INUSE == l1d.inuseFlag) {
+                    log_api::error("Switching to another user is not allowed while there are open L1 descriptors.");
+                    return SYS_NOT_ALLOWED;
+                }
+            }
```

Comment:
Consider making this an option.

Behavior of option:
- Return SYS_NOT_ALLOWED if there are open l1 descriptors
- Or, close open l1 descriptors

Proposed names for option:
- close_all_open_file_descriptors
- close_any_open_file_descriptors
- close_any_open_replicas (second/third choice)
- close_all_open_replicas (second/third choice)
- close_open_file_descriptors
- close_open_l1_descriptors
- **close_open_replicas** (winner?)
- torch_open_l1_descriptors
- l1_descriptor_behavior
- open_l1_descriptor_behavior
- on_open_l1_descriptor

Should all options be defined via a key-value pair/condInput instead? If yes, then the names of the properties can be ...
- switch_proxy_user => `KW_SWITCH_PROXY_USER`
  - Defaults to No.
- close_svr_to_svr_connections => `KW_CLOSE_SVR_TO_SVR_CONNECTIONS`
  - Defaults to Yes.
- close_open_replicas => `KW_CLOSE_OPEN_REPLICAS`
  - Defaults to No.

Reply:
Done.